### PR TITLE
Detailpage  fetch styling with new designs

### PR DESCRIPTION
**Pull request by Hilal-Tapan**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -46,12 +46,29 @@ app.get('/sent', (req, res) => {
...
+    .eq('id', suggestionId)
+    .single();
+
+  if (error) {
+    console.error('Error fetching suggestion:', error);
+    // Handle the error appropriately, e.g., render an error page
+  } else {
+    console.log(suggestionData);
+    res.render('detailPage-1', {
+      title: 'detail',
```
Comment (RainbowJM) [inline app.js:65]: title value has to be with uppercase for the first letter

File: public/css/detail-page.css [hunk 2]
```diff
@@ -27,35 +27,44 @@ main {
 
 /* wish -----------------------------------------------------------------------------------------*/
 .wish {
-    background-color: var(--white);
     flex-direction: column;
     width: 100%;
-    padding: 3em;
     margin-bottom: 1.5em;
+    /* padding: ; */
```
Comment (RainbowJM) [inline public/css/detail-page.css:33]: if its not needed remove it

**Review by RainbowJM — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (Hilal-Tapan) [reply app.js:65]: done

**Review by RainbowJM — APPROVED** [review]
lgtm

**Pull request merged**